wire-ios-request-strategy/Sources/Request Strategies/Conversation/Actions/AddParticipantActionHandler.swift

```diff
@@ -193,7 +193,7 @@ class AddParticipantActionHandler: ActionHandler<AddParticipantAction> {
     }
 }
 
-extension AddParticipantActionHandler {
+extension Payload {
```

Comment:
I think the request and response payloads should be scoped locally so that we can maintain separation with the rest of the code base, if you need to use the same payload for a different action, create a new local structure in the `CreateGroupConversationActionHandler` just like we do here.

Reply:
I reverted the changes